content/kotlin/concepts/functions/extension-function.md

```diff
@@ -0,0 +1,25 @@
+## Extension Function 
+
+- These functions provide ease to add more functionality to the existing classes, without inheriting them. This is called an Extension Function. 
+
+- Below example demonstrates the use of this concept : 
```

Comment:
In the other sections of this entry the author has done a good job of explaining the different elements and options for the syntax. That's missing here, there should be more information about how extension functions work. Instead of just presenting an example there should be details about what are the key syntax elements, and any other information that would be helpful.